index.js

```diff
@@ -72,7 +74,16 @@ class Conf {
 	}
 	get store() {
 		try {
-			return Object.assign(obj(), JSON.parse(fs.readFileSync(this.path, 'utf8')));
+			let data = fs.readFileSync(this.path, 'utf8');
+
+			// If `options.cryptoKey` is given, decrypt data.
+			if (this.cryptoKey) {
+				const decipher = crypto.createDecipher('aes-256-cbc', this.cryptoKey);
```

Comment:
Why did you pick `'aes-256-cbc'`?

Reply:
Just one that I was familiar with - we could include the encryption method as an option if you'd like. 

https://nodejs.org/api/crypto.html#crypto_crypto_createcipher_algorithm_password

Reply:
In regards to this, we could change the options from:

    new Conf({ encryptionKey: 'my_key' });

To something like:

    new Conf({ encryption: { key: 'my_key', algorithm: 'aes-256-cdc' });

That way the user can choose the algorithm. Since [the algorithm is dependent on OpenSSL](https://nodejs.org/api/crypto.html#crypto_crypto_createcipher_algorithm_password) it may be useful for some systems whose OpenSSL doesn't have `aes-256-cbc`.

---

And if `encryption` is just a string, then fallback to just using that as the encryption key with a sensible default.